inputs/torus.pin

```diff
@@ -137,3 +147,10 @@ rin = 6.0
 rmax = 12.0
 n_inside_horizon = 5
 magnetized = true
+entropy = 4
+
+# For tabulated EoS
+<units>
+scale_free = false
+geom_mass_msun = 2.8
+fluid_density_cgs = 1.e11
```

Comment:
This probably will cause tests to fail since I think they're based on ideal gas torus. Maybe we need two input decks?

Reply:
Good catch. I meant to comment that out. I think it's probably fine to leave it as 
```
#<units>
#scale_free = false
...
```
What do you think?

Reply:
yeah it's fine to just comment out the nuclear EOS stuff. Better in fact, so it's easy to see how to swap between modes

Reply:
For that matter I meant to do the same with the `<eos>` block.